jetty-start/src/main/java/org/eclipse/jetty/start/FileInitializer.java

```diff
@@ -128,7 +128,7 @@ protected Path getDestination(URI uri, String location) throws IOException
     protected void download(URI uri, Path destination) throws IOException
     {
         if (FS.ensureDirectoryExists(destination.getParent()))
-            StartLog.log("MKDIR", _basehome.toShortForm(destination.getParent()));
+            StartLog.log("INFO", "mkdir " + _basehome.toShortForm(destination.getParent()));
```

Comment:
I kind of liked the previous format, although it was not consistent (e.g. was using `INFO` for module activity).

If you go this direction, then there are 3 occurrences of `StartLog.log("COPY", ...)` and 1 of `StartLog.log("DOWNLD", ...)` to replace.

Reply:
I'm torn.... but on balance I think that INFO/WARN are least surprise.